node-red-node-wot/plugin-resources-src/node-wot-plugin-lib.ts

```diff
@@ -0,0 +1,954 @@
+const PROTOCOLS = ["http", "ws", "coap", "mqtt", "opcua", "modbus"]
+const DATATYPES = {
```

Comment:
this is the really useful part. Not in this PR but later on this can be made configurable by the user